fury/utils.py

```diff
@@ -1073,3 +1073,21 @@ def get_bounds(actor):
 
     """
     return actor.GetMapper().GetInput().GetBounds()
+
+
+def opacity(actor, opacity_level=0.5):
```

Comment:
not sure it is needed. Otherwise, this function needs a test.

Reply:
I agree. It will be beneficial if we can set the opacity of individual objects instead.